### DISCUSSION TITLE
Web SDK 在移动端H5的兼容性怎么样

### DISCUSSION THREAD

**Discussion by suhuamin** [Tencent/libpag#58] [Q&A] [answered]
你好，我运行了demo，在移动端运行没什么问题。想问一下在移动端一些复杂的动画场景下会不会存在兼容性问题，因为看到我们的文档只提及了 Chrome和Safari浏览器，没有提及移动端的兼容性

**Comment by zenoslin** [comment, marked as answer]
移动端浏览器的兼容性我们正在测试中，还没结束。
不能排除复杂动画场景下存在兼容性问题的情况，当然发现兼容性问题的话，我们会快速修复。
![image](https://user-images.githubusercontent.com/24915910/150497717-30b496bf-150e-414c-9c62-55518bb433be.png)
像以上提到的，移动端微信浏览器的场景下存在的问题，我们也在寻求方案解决中。
